proto/frequenz/api/dispatch/v1/dispatch.proto

```diff
@@ -151,17 +151,68 @@ message DispatchDetail {
 
 // Filter parameter for specifying multiple time intervals
 message TimeIntervalFilter {
-  // Filter by start_time >= this timestamp
-  google.protobuf.Timestamp start_from = 1;
+  // Filter by time >= this timestamp.
+  google.protobuf.Timestamp from = 1;
 
-  // Filter by start_time < this timestamp
-  google.protobuf.Timestamp start_to = 2;
+  // Filter by time < this timestamp.
+  google.protobuf.Timestamp to = 2;
+}
```

Comment:
estimated_end_time will be calculated for all dispatches - not only reoccurring
So it make sense to leave the end_to, end_from here. Please look at:
https://github.com/frequenz-floss/frequenz-api-dispatch/pull/186/commits/92f675dd24686ee3b874862985992fff00043c06

Reply:
It's not removed, just spilt up into start interval and end interval, see: https://github.com/frequenz-floss/frequenz-api-dispatch/pull/184/files#diff-b2f8ee9bf7dcd2f8c9a8d9434771f0229aca74577fdda0831e26be679fae6cdbR212

Reply:
Oww that is right, sorry for misleading comment . 
For some reason I though it is now part of RecurrenceFilter, but it is not ...
Anyway approve :)